README.md

```diff
@@ -191,7 +191,7 @@ Example:
 #### cache_has_many
 
 Options:
-_[:embed]_ When true, specifies that the association should be included with the parent when caching. This means the associated objects will be loaded already when the parent is loaded from the cache and will not need to be fetched on their own. When :ids, only the id of the associated records will be included with the parent when caching.
+_[:embed]_ When true, specifies that the association should be included with the parent when caching. This means the associated objects will be loaded already when the parent is loaded from the cache and will not need to be fetched on their own. When :ids, only the id of the associated records will be included with the parent when caching. Defaults to `true`.
```

Comment:
The default is `:ids`?

Reply:
Oops.  Thanks for catching that.  Pushed a fix